### PR TITLE
add TopBar changes, added AccountDetails

### PR DESCRIPTION
**Pull request by andrewdamelio**
#3917, #3919, #3925 #3922

### CODE REVIEW

File: packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts [hunk 1]
```diff
@@ -1,67 +1,61 @@
...
-    return "ropsten";
-  } else if (networkId === "4") {
-    return "rinkeby";
-  } else if (networkId === "42") {
-    return "kovan";
-  } else if (networkId === "104") {
-    // TODO: Formatic currently doesn't support local nodes
-    // TODO: Update this when they do
-    return false;
+  if (networkId === '1') {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts]: use NETWORK_IDS in constants

File: packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts [hunk 2]
```diff
@@ -1,67 +1,61 @@
...
 
   const networkId: string = getNetworkId();
-  const supportedNetwork: string | false = getFormaticNetwork(networkId);
+  const supportedNetworks = getFormaticNetwork(networkId);
 
-  if (supportedNetwork) {
+  if (supportedNetworks) {
     try {
-      const fm = new Fortmatic(networkId === "1" ? FORTMATIC_API_KEY : FORTMATIC_API_TEST_KEY, supportedNetwork);
+      const fm = new Fortmatic(networkId === '1' ? FORTMATIC_API_KEY : FORTMATIC_API_TEST_KEY, supportedNetworks);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts]: use NETWORK_IDS in constants

File: packages/augur-ui/src/modules/auth/actions/login-with-portis.ts [hunk 3]
```diff
@@ -1,63 +1,78 @@
...
   const myPrivateEthereumNode = {
-    nodeUrl: "http://localhost:8545",
+    nodeUrl: 'http://localhost:8545',
     chainId: networkId,
   };
-  if (networkId === "1") {
-    return "mainnet";
-  } else if (networkId === "42") {
-    return "kovan";
+  if (networkId === '1') {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/actions/login-with-portis.ts]: use NETWORK_IDS in constants

File: packages/augur-ui/src/modules/auth/actions/logout.ts [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
-import { clearUserTx } from "modules/contracts/actions/contractCalls";
+import { clearLoginAccount } from 'modules/account/actions/login-account';
+import { clearUserTx } from 'modules/contracts/actions/contractCalls';
+import { ThunkDispatch } from 'redux-thunk';
+import { Action } from 'redux';
+import { windowRef } from 'utils/window-ref';
 
 export function logout() {
-  return (dispatch: ThunkDispatch<void, any, Action>) => {
+  return async (dispatch: ThunkDispatch<void, any, Action>) => {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/actions/logout.ts:8]: I don't see `await` do we need `async`?

File: packages/augur-ui/src/modules/modal/containers/modal-signin.ts [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+export default withRouter(
+  connect(
+    mapStateToProps,
+    mapDispatchToProps,
+    mergeProps
+  )(SignIn)
+);
+
+// If required, follow any instructions in the ${accountType} window.
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/containers/modal-signin.ts:166]: can this commented out code be removed?

Comment (andrewdamelio) [reply packages/augur-ui/src/modules/modal/containers/modal-signin.ts:166]: Opps - good catch

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💚

 This link will always point to this PR's latest changes: https://peek.run/w9jhlv3

**Review by bthaile — CHANGES_REQUESTED** [review]
looks great, some nit pics

**Review by bthaile — APPROVED** [review]
lgtm, need to update loginAccount `meta` in types.ts  that's a super nit pic, approving

**Pull request merged**